pysc2/pysc2/agents/random_agent.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path'); //eslint-disable-line
+const base_agent = require(path.resolve(__dirname, './base_agent.js'))
+const actions = require(path.resolve(__dirname, '..', 'lib', './action.js'))
+const pythonUtils = require(path.resolve(__dirname, '..', 'lib', './.pythonUtils.js'))
+const randomChoice = pythonUtils
```

Comment:
const { randomChoice } = pythonUtils

if you're curious about this syntax read about object unpacking (destructuring) in javascript
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
still gotta fix this

Reply:
It has been fixed.